homeassistant/components/sensibo/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
```

Comment:
```py
        self._attr_temperature_unit = (
            UnitOfTemperature.CELSIUS
            if self.device_data.temp_unit == "C"
            else UnitOfTemperature.FAHRENHEIT
        )
```
Can be removed from climate.py

Reply:
No, why would you say that?

Reply:
Because you already overwrite the temperature unit property

Reply:
Yes, you're right. That's going away

Reply:
Prel PR: https://github.com/home-assistant/core/pull/134412

Reply:
Prel PR: https://github.com/home-assistant/core/pull/134412